### PR TITLE
STOR-1577: Rebase to v1.7.3 for OCP 4.16

### PR DESCRIPTION
**Pull request by RomanBednar**
Diff to upstream v1.7.3:
https://github.com/kubernetes-sigs/aws-efs-csi-driver/compare/v1.7.3...RomanBednar:rebase-v1.7.3

Full changelog: https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/master/CHANGELOG-1.x.md

@openshift/storage

### CODE REVIEW

File: .snyk [hunk 1]
```diff
@@ -1,6 +0,0 @@
-# References:
```
Comment (jsafrane) [inline .snyk:1]: Please do not remove .snyk file.

Comment (RomanBednar) [reply .snyk]: Added.

**Comment by ropatil010** [comment]
Added the results here: https://issues.redhat.com/browse/STOR-1577. 
/label qe-approved

**Pull request merged**